Plants/Weed.gd

```diff
@@ -40,14 +40,26 @@ func spread_impl(tiles):
 	)
 	# Create a new weed on the tiles which it is spreading to
 	for tile in toSpread:
-		if (tile.is_occupied() && tile.child_plant.status != Status.DEAD && killable_plants.has(tile.child_plant)):
-			tile.child_plant.kill()
-		else:
-			var weed = load("res://Plants/Weed.tscn").instantiate()
-			weed.sow(tile, false)
+		var weed = load("res://Plants/Weed.tscn").instantiate()
+		weed.sow(tile, false)
 
 func can_sow(tile : PlantTile, use_seed: bool = false) -> bool:
-	return !tile.has_adjacent(Global.PlantType.SPIKY_PLANT)
+	return !tile.has_adjacent(Global.PlantType.SPIKY_PLANT) || !tile.child_plant.plant_type() == Global.PlantType.SPIKY_PLANT
```

Comment:
Is that supposed to be && instead of ||?

(But also if the spiky plant is dead (which it has to be to be a child), maybe it should be allowed for it to be weedy)

Reply:
yes and no

Reply:
no and no